src/wp-includes/media.php

```diff
@@ -5455,6 +5455,20 @@ function wp_get_loading_attr_default( $context ) {
 		return false;
 	}
 
+	/*
+	 * Skip programmatically created images within post content as they need to be handled together with the other
+	 * images within the post content.
+	 * Without this clause, they would already be counted below which skews the number and can result in the first
+	 * post content image being lazy-loaded only because there are images elsewhere in the post content.
+	 */
+	static $special_content_contexts = array(
+		'the_post_thumbnail'      => true,
+		'wp_get_attachment_image' => true,
```

Comment:
Why do these need to have a value?

Reply:
See below, in order to use `isset`, which only works with array keys.